corehq/apps/reports/display.py

```diff
@@ -162,5 +162,5 @@ def xmlns_to_name_for_case_deletion(domain, form):
             "[Unknown Module]",
             form.name or "[Unknown Form]"
         ]
-        form_name = ' > '.join(extracted_name)
+        form_name = escape(' > '.join(extracted_name))
```

Comment:
This seems unnecessary and not the right place to be escaping for HTML. Does the template not do this escaping automatically when it renders the form name?

Reply:
Ahhh, ok so django templates do html escape by default. I agree this isn't the right place to escape but if the template escapes strings already is there a need to explicitly escape it here? I'm a little confused because I think you also suggested I do so a while back in the template itself (I used escapejs but it would escape all characters '>' characters which was a problem). Apologies for the late response also!!

Reply:
I think I used poor wording in my earlier suggestion to escape in the template. The template should escape HTML characters automatically (as long as it's not tagged with `mark_safe` or some variant), so there is usually no need to use `escapejs`.